.github/workflows/deploy-dev-green.yml

```diff
@@ -1,16 +1,16 @@
-name: Deploy to Dev-green (ECS)
+name: Deploy to Read-only (ECS)
 
 on: workflow_dispatch
 
 jobs:
   deploy:
     name: Deploy
     runs-on: ubuntu-latest
-    environment: dev-green
-    concurrency: dev-green
+    environment: read-only
+    concurrency: read-only
     env:
       ECS_CLUSTER: api
-      ECS_SERVICE: api-dev-green
+      ECS_SERVICE: api-read-only
```

Comment:
question: is this a permanent name change for this environment? We have an RFC that specifies how environments are supposed to be named now, and I'd like to stick to it if possible: https://github.com/mbta/technology-docs/blob/main/rfcs/accepted/0015-consistent-environment-naming.md. Also, the file is still called `deploy-dev-green.yml`.

Reply:
No, I'll remove this change before merging. The step needed to be in a file which exists on the default branch, so I couldn't create a new deploy file only for this PR. I'll also remove all the infrastructure for this `api-read-only` test environment.